Dashboard/Views/MainWindowView.axaml

```diff
@@ -22,7 +22,7 @@
             <Grid ColumnDefinitions="0.8*, 0.2*" Margin="10, 0">
                 <TextBlock Grid.Column="0" FontWeight="Black" FontFeatures="Bold"
                            FontSize="25" VerticalAlignment="Center">
-                    Nuvotion FSAE-Scrutineering Tool
+                    Formula Autonomous Vehicle Scrutineering Tool
```

Comment:
Might want to change the window title too?

Reply:
25e7e8b17b8cb3e65cf1fcefa4278b5b7514f39a